src/main/java/org/isf/medicalinventory/model/MedicalInventoryRow.java

```diff
@@ -76,10 +76,7 @@ public class MedicalInventoryRow extends Auditable<String> {
 	@ManyToOne
 	@JoinColumn(name="MINVTR_LT_ID_A")
 	private Lot lot;
-	
```

Comment:
If you are removing a column from the object doesn't there need to be some corresponding changes to the SQL files?